doc/Quick-Start-Guide.rst

```diff
@@ -66,11 +66,11 @@ Perform the below mentioned procedure to build the source code.
    - **$ scripts/m0 rebuild**
 
    **Note**: The **scripts/m0 rebuild** command includes some clean up.
-   
+ 
 RPM Generation
 ===============
 
-If you want to make sure about generating RPMs, run the below mentioned command.
+You can build RPMs instead of building from source. Do not do this if you have followed the above steps for build from source.
```

Comment:
This is still misleading. Building binary executables and building RPMs are orthogonal. 
Users can do either of them, or both of them.
So, I would like to state this as:
```
You can also build RPMs by running the following command.
```

Reply:
Yes but I have found that you get conflicts when you do both, which causes an error.

Reply:
I have changed to your suggestion.